ev3dev/motor.py

```diff
@@ -1695,3 +1699,199 @@ def on_for_seconds(self, steering, speed_pct, seconds, brake=True, block=True):
     def on(self, steering, speed_pct):
         (left_speed_pct, right_speed_pct) = self.get_speed_steering(steering, speed_pct)
         MoveTank.on(self, left_speed_pct, right_speed_pct)
+
+
+class MoveJoystick(MoveTank):
+    """
+    Used to control a pair of motors via a joystick
+    """
+
+    def angle_to_speed_percentage(self, angle):
```

Comment:
I think we should be referring to this as "arcade drive".

Reply:
But it takes an angle and returns speed percentages for the two motors...the current name seems much more descriptive to me.

Reply:
I'm proposing that you rename the class and any other references to the concept. There are many ways to drive some motors with joysticks, so being specific to make it clear what algorithm is being used would be helpful.

Reply:
How about `MoveJoystickArcade`?

Reply:
Where does 'Arcade' come from? Is it a common term for this? Frankly, neither of the names make sense to me and I would have to read the comments anyway to understand the purpose of this function.

Reply:
It is just because a lot of arcade machines have joysticks that you use to control vehicles, characters, etc.  What if we included the name Tank in the class....`MoveTankJoystick`?  Are there other ways one might use a joystick to move a tank?

Reply:
I cleaned up the angle compares, haven't changed the name of the class yet but I am game for changing it, we just need to pick a name.